form_components/TreatmentForm.js

```diff
@@ -57,23 +57,31 @@ function TreatmentForm({ addTreatments }){
 const styles = StyleSheet.create({
     container: {
         backgroundColor: GlobalColors.backgroundColor,
-        flex: 1,
-        justifyContent: 'center',
-        alignContent: 'center'
+        alignContent: 'center',
+        flex:1,
+        paddingTop:100
     },
     textInputBox: {
         borderWidth: 1,
         padding: 10,
         marginBottom: 5,
         fontSize: 18,
         borderRadius: 6,
-        backgroundColor: '#EDEFEF'
+        backgroundColor: '#EDEFEF',
```

Comment:
Same comment about GlobalColors.

```suggestion
        backgroundColor: GlobalColors.backgroundColor,
```